### PR TITLE
Update documentation for container monitoring addon

### PR DESCRIPTION
**Pull request by rashmichandrashekar**
Fixing documentation for container monitoring addon.

### CODE REVIEW

File: docs/tutorials/containermonitoringaddon.md [hunk 1]
```diff
@@ -9,7 +9,8 @@ Container health gives you performance monitoring ability by collecting memory a
 
 # Prerequisites!
 
-  - A Log Analytics workspace. You can create it when you enable monitoring of your new AKS cluster or let the onboarding experience create a default workspace in the default resource group of the AKS cluster subscription. If you chose to create it yourself, you can create it through [Azure Resource Manager], through [PowerShell], or in the [Azure portal].
+  - A Log Analytics workspace. You can create it when you enable monitoring of your new AKS cluster. You can create it through [Azure Resource Manager], through [PowerShell], or in the [Azure portal].
```
Comment (jackfrancis) [inline docs/tutorials/containermonitoringaddon.md]: Should we change "your new AKS cluster" to "your new AKS Engine cluster"?

Comment (ganga1980) [reply docs/tutorials/containermonitoringaddon.md]: removed this section so this comment not applicable now.

File: examples/addons/container-monitoring/README.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+      - kubectl delete -f <kubernetesmasteraddons-omsagent-daemonset.yaml>
+
+        or
+
+      - deleting omsagent daemonset and replicaset using Kubectl tool
+
+## Contact
+
+If you have any questions or feedback regarding the container monitoring add-on, please reach us out thru [this](mailto:askcoin@microsoft.com) email.
```
Comment (CecileRobertMichon) [inline examples/addons/container-monitoring/README.md]: s/thru/through ?

Comment (CecileRobertMichon) [reply examples/addons/container-monitoring/README.md]: s/add-on/addon

File: examples/addons/container-monitoring/README.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+
+1. [Add the 'AzureMonitor-Containers' Solution to your Log Analytics workspace.](http://aka.ms/coinhelmdoc)
+
+2. [For AKS-engine K8S cluster, add clusterName and Log Analytics workspace tag to cluster resources, to be able to use Azure Container monitoring User experience (aka.ms/azmon-containers)](http://aka.ms/coin-acs-tag-doc)
+
+> Note: At the moment, if the AKS engine version is v0.29.1 or ACS engine Kubernetes cluster, default clusterName is my_acs_cluster_name in the container monitoring add-on. In  AKS-engine version greater than v0.29.1, the default cluster name is aks-engine-cluster. This name needs to be tagged as clusterName value in step #2 above.
+
+Navigate to [azmon-containers](https://aka.ms/azmon-containers) to view the health, metrics and logs of your Kubernetes cluster(s).
+
+If you have any questions regarding the Container Monitoring add-on, please reach us out thru [this](mailto:askcoin@microsoft.com) email.
```
Comment (CecileRobertMichon) [inline examples/addons/container-monitoring/README.md]: s/thru/through

File: docs/topics/monitoring.md [hunk 4]
```diff
@@ -54,6 +54,29 @@ NAME            CPU(cores)   MEMORY(bytes)
 somePod         0m           11Mi
 ```
 
+## Azure Monitor for containers
+
+Azure Monitor for containers is a feature designed to monitor the performance of container workloads deployed to AKS-engine (formerly known as ACS-engine) cluster(s) hosted on Azure. Monitoring your containers is critical, especially when you're running a production cluster, at scale, with multiple applications.
```
Comment (CecileRobertMichon) [inline docs/topics/monitoring.md]: let's standardize the spelling of `AKS-Engine` everywhere in this doc

Comment (CecileRobertMichon) [reply docs/topics/monitoring.md]: (same for ACS-Engine)

**Comment by CecileRobertMichon** [comment]
Let's fix the commit message before merging.

Also added a few spelling comments.

/hold

File: docs/topics/monitoring.md (hunk 4 above)
Comment (ganga1980) [reply docs/topics/monitoring.md]: @CecileRobertMichon, I see the name is not consistent across the docs. What is the standardize name we should update for AKS Engine and ACS engine in our docs? should we go with AKS-Engine and ACS-Engine?

Comment (CecileRobertMichon) [reply docs/topics/monitoring.md]: correct, we're trying to get to a consistent state and converge to `AKS-Engine` and `ACS-Engine` (same as they appear in the README)

File: examples/addons/container-monitoring/README.md (hunk 2 above)
Comment (ganga1980) [reply examples/addons/container-monitoring/README.md]: For Addon also, there is inconsistent usage (add-on, addon) across the docs.  I believe, at some point, everything else will be updated to use "addon" so I can update to addon in my doc updates?

Comment (CecileRobertMichon) [reply examples/addons/container-monitoring/README.md]: correct

Comment (ganga1980) [reply examples/addons/container-monitoring/README.md]: thanks. fixed.

File: examples/addons/container-monitoring/README.md (hunk 3 above)
Comment (ganga1980) [reply examples/addons/container-monitoring/README.md]: Thanks fixed.

File: docs/topics/monitoring.md (hunk 4 above)
Comment (ganga1980) [reply docs/topics/monitoring.md]: updated as discussed all the changes. @CecileRobertMichon, please have a look at them.

File: docs/topics/monitoring.md [hunk 5]
```diff
@@ -54,6 +54,29 @@ NAME            CPU(cores)   MEMORY(bytes)
...
+
+Azure Monitor for containers gives you performance visibility by collecting memory and processor metrics from controllers, nodes, and containers that are available in Kubernetes through the Metrics API. Container logs are also collected. After you enable monitoring from Kubernetes clusters, these metrics and logs are automatically collected for you through a containerized version of the Log Analytics agent for Linux and stored in your Log Analytics workspace.
+
+Azure Monitor for containers for AKS-Engine cluster(s) can be configured through following options
+
+1. Helm chart [azuremonitor-containers](https://github.com/helm/charts/tree/master/incubator/azuremonitor-containers)
+2. [Container-monitoring add-on](../../examples/addons/container-monitoring/README.md)
+
+> Note: If more than one AKS engine cluster planned to configure to the same Azure Log Analytics Workspace then recommend option is to use Helm chart (i.e. option #1 above)
```
Comment (CecileRobertMichon) [inline docs/topics/monitoring.md]: one more AKS engine --> `AKS-Engine` reference

Comment (ganga1980) [reply docs/topics/monitoring.md]: fixed :)

**Comment by rashmichandrashekar** [comment]
Thanks @ganga1980 for fixing the comments!

**Pull request merged**